content/en/docs/concepts/configuration/secret.md

```diff
@@ -337,16 +326,30 @@ A container using a Secret as a
 Secret updates.
 {{< /note >}}
 
-### Using Secrets as environment variables
+##### Automatic mounting of Secrets
+
```

Comment:
Could you find a way to place this section into another section to avoid level H5?

Reply:
What might work, but I'm wary of the amount of effort: extracting the details of the mechanism that are common to ConfigMap and Secret, and making those have their own concept page.

WDYT?